frontend/test/__support__/cypress.js

```diff
@@ -199,3 +199,29 @@ export function createBasicAlert({ firstAlert, includeNormal } = {}) {
   cy.findByText("Done").click();
   cy.findByText("Let's set up your alert").should("not.exist");
 }
+
+// Functions specific to QA databases that we started supporting recently in CI
+export function addPostgresDatabase(db_display_name = "QA Postgres12") {
```

Comment:
I tried to use API for this, but I'm getting `invalid length of startup packet` for postgres. Using API for Mongo, OTOTH works without a hiccup.